src/main/java/ScrabbleErrorMessages.java

```diff
@@ -0,0 +1,17 @@
+public enum ScrabbleErrorMessages
```

Comment:
Constants. Not enum constants.
Making enums out of just strings is overkill.

Reply:
If I make final constants, IntelliJ complains that it should be local fields to the playWord method

Reply:
Show me what you had

Reply:
I wasn't using the word static, I think that's what caused it. I added static this time